lib/elixir/lib/calendar.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Calendar do
```

Comment:
Thank you, I will add the deprecation warning so you can safely remove it in the future from your library.